grape/lib/task.rb

```diff
@@ -6,11 +6,14 @@ def initialize(dictionary)
     @date = dictionary[:date]
     @people_left = dictionary[:people_left]
     @category = dictionary[:category]
+  end
 
-    validate_properties
+  def valid?
+    return false if [@title, @date, @people_left, @category].include?(nil)
+    true
   end
 
-  def validate_properties
-    raise ArgumentError if [@title, @date, @people_left, @category].include?(nil)
+  def future?
```

Comment:
No need to do anything now, but just think that the app and the server may have different timezones and how to deal with the problem